Paintroid/src/main/java/org/catrobat/paintroid/contract/MainActivityContracts.kt

```diff
@@ -174,7 +174,7 @@ interface MainActivityContracts {
 
     interface Interactor {
         fun saveCopy(callback: SaveImageCallback, requestCode: Int, workspace: Workspace)
-        fun createFile(callback: CreateFileCallback, requestCode: Int, filename: String)
+        fun createFile(callback: CreateFileCallback, requestCode: Int, filename: String?)
```

Comment:
Why?
Should it be possible to create a file without a name?

Reply:
ah.. actually it's given a null value in MainActivity line 249 and at 2 places in MainActivityPresenterTest, so to accommodate that I had added the `?` here, but I just noticed that this function is called only when the name is not null, so I think I can safely remove it.